$OpenBSD: patch-chrome_browser_printing_print_system_task_proxy_cc,v 1.4 2012/10/12 16:46:09 robert Exp $

Fix build with cups >= 1.6
error: 'ppd_attr_t* ppdFindAttr(ppd_file_t*, const char*, const char*)' is deprecated

--- chrome/browser/printing/print_system_task_proxy.cc.orig	Tue Sep 25 15:02:11 2012
+++ chrome/browser/printing/print_system_task_proxy.cc	Wed Oct 10 17:19:28 2012
@@ -421,10 +421,7 @@ bool PrintSystemTaskProxy::ParsePrinterCapabilities(
       }
     }
 
-    bool is_color_device = false;
-    ppd_attr_t* attr = ppdFindAttr(ppd, kColorDevice, NULL);
-    if (attr && attr->value)
-      is_color_device = ppd->color_device;
+    bool is_color_device = ppd->color_device;
     *set_color_as_default = is_color_device;
 
     if (!((is_color_device && getBasicColorModelSettings(
